Filters/CDR0000482943.xml

```diff
@@ -27,28 +27,14 @@
  <!--
  =================================================================
  Template to denormalize the Summary elements
- Ignore errors if the filter is run as a PublishPreview or QC
- report.
  ================================================================= -->
  <xsl:template                   match = "Summary">
   <xsl:variable                   name = "docID"
                                 select = "@cdr:ref"/>
 
-  <xsl:variable                   name = "docInfo">
```

Comment:
I need to look back at our discussion regarding the use or 'cdr:' vs 'cdrx:' but this change looks like we now require a publishable version to exist for a PublishPreview report when before we didn't.

Reply:
If you think that based on the prefix ("cdr") for the linking attribute, you want to remember that this prefix is not carrying any meaning, the way the scheme names "cdr" and "cdrx" do. You can think of it as a coincidence that the first scheme uses the same string its name as the namespace prefix does.

Reply:
My remark was actually related to the content of the variable 'docID'.  We had a choose block based on the content of the parameter "isPP" which determined if we were calling the document() function with a URI starting with 'cdr:...' or 'cdrx:...'